Makefile

```diff
@@ -71,4 +71,15 @@ clean:
 	-vagrant destroy -f
 	-rm -r .vagrant
 
+# Generate markdown compatibility matrix
+markdown-matrix:
+	@ansible -i hosts --list-hosts all | \
+		sort | \
+		grep -v tester-es | \
+		grep -v '^#' | \
+		grep tester | \
+		sed 's#tester-##g' | \
+		tr -d " " | \
+		sed 's#\(.*\)#\1 | :white_check_mark:#g'
```

Comment:
make goal to generate the markdown matrix details that are shown in the README.md